examples/shopping-cart/components/ProductList.vue

```diff
@@ -17,13 +17,20 @@ import { mapGetters, mapActions } from 'vuex'
 
 export default {
   computed: mapGetters({
-    products: 'allProducts'
+    products: 'product/list'
   }),
-  methods: mapActions([
-    'addToCart'
-  ]),
+  methods: {
+    ...mapActions({
+      addItemToCart: 'cart/addItem',
+      decreaseProductInventory: 'product/decreaseInventory'
+    }),
+    addToCart(product) {
+      this.addItemToCart(product);
+      this.decreaseProductInventory(product);
+    }
+  },
   created () {
-    this.$store.dispatch('getAllProducts')
+    this.$store.dispatch('product/getAll')
```

Comment:
If you already map actions, you can also map this one and simply call `this.getAllProducts()` here. What do you think?